creating-workflows/src/main/java/org/springframework/cloud/appbroker/samples/creatingworkflows/AppBrokerApplication.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2002-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.appbroker.samples.creatingworkflows;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SuppressWarnings({"PMD.UseUtilityClass"})
```

Comment:
hmm. that's annoying to have to add this.

Reply:
The alternative was to ignore this rule. I don't have strong feelings either way.